lib/_included_packages/plexnet/video.py

```diff
@@ -452,6 +455,12 @@ def genres(self):
     def roles(self):
         return self.show().roles
 
+    @property
+    def intro(self):
+        if not self._intro:
+            self._intro = (filter(lambda x: x.type == "intro", self.markers) or [None])[0]
```

Comment:
So we don't set this repeatedly if there aren't any markers.
```suggestion
        if self._intro is None:
            self._intro = (filter(lambda x: x.type == "intro", self.markers) or [False])[0]
```